langchain/callbacks/tracers/langchain.py

```diff
@@ -44,8 +75,24 @@ def _get_tenant_id(
         return tenant_id_
     endpoint_ = endpoint or get_endpoint()
     headers_ = headers or get_headers()
-    response = requests.get(endpoint_ + "/tenants", headers=headers_)
-    raise_for_status_with_text(response)
+    response = None
+    try:
+        response = requests.get(endpoint_ + "/tenants", headers=headers_)
```

Comment:
Shouldn't this be removed, or is that another PR?

Reply:
https://github.com/hwchase17/langchain/pull/5198